src/dev/Web/EastSeat.ResourceIdea.Web/Components/Shared/SidebarNavigation.razor

```diff
@@ -0,0 +1,111 @@
+@* ----------------------------------------------------------------------------------
+    File: SidebarNavigation.razor
+    Path: src/Web/EastSeat.ResourceIdea.Web/Components/Shared/SidebarNavigation.razor
+    Description: Vertical sidebar navigation component with collapsible sections
+   ---------------------------------------------------------------------------------- *@
+
+<div class="sidebar" id="sidebar">
+    <!-- Brand Header -->
+    <div class="sidebar-header">
+        <div class="brand">
+            <div class="brand-logo">
+                <img src="/images/resourceidea-logo.svg" alt="ResourceIdea Logo" style="height: 32px; width: auto;" />
+            </div>
+            <span class="brand-text">ResourceIdea</span>
+        </div>
+    </div>
+
+    <!-- Search Bar -->
+    <div class="sidebar-search">
+        <div class="search-container">
+            <i class="fas fa-search search-icon"></i>
+            <input type="text" class="search-input" placeholder="Search" />
+        </div>
+    </div>
+
+    <!-- Navigation Menu -->
+    <div class="sidebar-menu">
+        <!-- Planner Section -->
+        <div class="menu-section">
+            <div class="menu-section-header" @onclick="TogglePlannerSection">
+                <span class="menu-section-title">Planner</span>
+                <i class="fas fa-chevron-right section-toggle @(isPlannerExpanded ? "expanded" : "")"></i>
+            </div>
+            <div class="menu-section-content @(isPlannerExpanded ? "expanded" : "")">
+                <NavLink class="menu-item" href="/service-lines-timeline">
+                    <i class="fas fa-chart-line"></i>
+                    <span>Service Lines Timeline</span>
+                </NavLink>
+                <NavLink class="menu-item" href="/job-managers-timeline">
+                    <i class="fas fa-users-cog"></i>
+                    <span>Job Managers Timeline</span>
+                </NavLink>
+            </div>
+        </div>
+
+        <!-- Clients Section -->
+        <div class="menu-section">
+            <div class="menu-section-header" @onclick="ToggleClientsSection">
```

Comment:
Refactor so that if one of the sections is expanded, the other sections are collapsed. This should be done for the planner, clients and administration sections.